policyengine_us/parameters/gov/states/nc/ncdhhs/scca/age/group.yaml

```diff
@@ -2,25 +2,26 @@ description: North Carolina assigns children to these age groups for Subsidized
 
 brackets:
   - threshold:
-      2022-10-01: 0
+      2022-10-01: 0 # Infant
     amount:
-      2022-10-01: 1  # Infant
+      2022-10-01: 1  
   - threshold:
-      2022-10-01: 1
+      2022-10-01: 1.01 # 2-year-olds
```

Comment:
Make age 1.01 here, so the 12 months infants in group 1. Please let me know if it's the ideal way.